protocol/driver.go

```diff
@@ -20,32 +20,49 @@ import (
 
 //go:generate protoc --proto_path=$GOPATH/src:. --gogo_out=plugins=grpc:. ./driver.proto
 
+const (
+	mb = 1024 * 1024
+
+	// DefaultGRPCMaxMsgSize is maximum msg size for gRPC (in bytes).
```

Comment:
Alternatively you can just put the units in the name: `DefaultGRPCMaxMessageBytes`